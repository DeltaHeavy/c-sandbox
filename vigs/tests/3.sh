#!/bin/bash

echo -n "Test 3: ftable........."
./ftable tests/3.in tests/3.testout
diff tests/3.testout tests/3.out >/dev/null 2>&1
ret=$?
rm tests/3.testout
if [ $ret -ne 0 ]
then
    echo -e "\033[31mFAIL\033[0m"
    false
else 
    echo -e "\033[32mPASS\033[0m"
    true
fi
