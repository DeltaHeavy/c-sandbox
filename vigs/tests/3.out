Total chars: 13
A:         0 (  0.00%) 
B:         2 ( 15.38%) ****************
C:         0 (  0.00%) 
D:         0 (  0.00%) 
E:         2 ( 15.38%) ****************
F:         0 (  0.00%) 
G:         0 (  0.00%) 
H:         0 (  0.00%) 
I:         0 (  0.00%) 
J:         0 (  0.00%) 
K:         0 (  0.00%) 
L:         0 (  0.00%) 
M:         0 (  0.00%) 
N:         1 (  7.69%) ********
O:         4 ( 30.77%) *******************************
P:         0 (  0.00%) 
Q:         0 (  0.00%) 
R:         1 (  7.69%) ********
S:         0 (  0.00%) 
T:         3 ( 23.08%) ************************
U:         0 (  0.00%) 
V:         0 (  0.00%) 
W:         0 (  0.00%) 
X:         0 (  0.00%) 
Y:         0 (  0.00%) 
Z:         0 (  0.00%) 
